pkg/apis/pipeline/v1alpha1/cluster_resource.go

```diff
@@ -118,12 +118,9 @@ func (s *ClusterResource) GetURL() string {
 	return s.URL
 }
 
-// GetParams returns the resoruce params
+// GetParams returns the resource params
 func (s ClusterResource) GetParams() []Param { return []Param{} }
 
-// GetSecrets returns an array with field name and the corresponding secret to populate it
-func (s ClusterResource) GetSecrets() []SecretParam { return s.Secrets }
```

Comment:
Isn't it there to satisfy an interface though ? :thinking:

Reply:
Maybe at one point it was? I can't find any interfaces that require it though :(

And it looks like it builds without it, so 🤷‍♂️ ?

Reply:
> And it looks like it builds without it, so man_shrugging ?

I accept that :100: